content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -195,6 +197,17 @@ To encrypt the data:
              timeout: 3s
    ```
 
+Setting `--encryption-provider-config-automatic-reload` to `true` collapses all health checks to a single health check endpoint. Individual health checks are only available when KMS v1 providers are in use and the encryption config is not auto-reloaded.
+
+Following table summarizes the health check endpoints for each KMS version:
```

Comment:
Below the table, provide examples of what `Individual Healthchecks` and `Single Healthcheck` look like, i.e. `/healthz/foo`.